app/src/components/cards/DeployDetailsCard/DeployDetailsCard.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { DeployStatus } from 'src/api/types';
+import { render } from '../../../test-utils';
+import { DeployDetailsCard } from './DeployDetailsCard';
+
+const mockDeploy = {
+  timestamp: 0,
+  timeSince: '',
+  readableTimestamp: '',
+  deployHash: 'testDeployHash',
+  blockHash: 'testBlockHash',
+  publicKey: 'testPublicKey',
+  status: DeployStatus.Success,
+  amount: '',
+  action: '',
+  deployType: '',
+  paymentAmount: '',
```

Comment:
Why are many of these values empty strings?

Reply:
the card only uses 3 of the values (deployHash, public key, and block hash). The blank values are all vals that are not used in the card